lib/sandbox/pmapi.js

```diff
@@ -119,7 +119,15 @@ function Postman (execution, onRequest, onAssertion, cookieStore, options = {})
              * @type {string}
              * @instance
              */
-            requestId: execution.legacy._itemId
+            requestId: execution.legacy._itemId,
+
+            /**
+             * The active request's current path, which reflects the hierarchical structure.
```

Comment:
This can be simplified.
```suggestion
     * The path of the request being run.
```